tests/api2/activedirectory.py

```diff
@@ -462,129 +408,96 @@ def test_45_re_enable_activedirectory(request):
+    results = POST('/filesystem/stat/', f'{SMB_PATH}/testdir/testfile2.txt')
+    assert results.status_code == 200, results.text
 
 
-@osx_host_cfg
-def test_54_Create_file_on_SMB_share_via_OSX_to_test_permissions(request):
+def test_46_delete_testfile_on_the_active_directory_share(request):
     depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
-    results = SSH_TEST('touch "%s/testfile.txt"' % MOUNTPOINT,
-                       OSX_USERNAME, OSX_PASSWORD, OSX_HOST)
+    command = fr'smbclient //{ip}/{SMB_NAME} -U {CMD_AD_USER}%{ADPASSWORD}' \
+        r' -m NT1 -c "rm testfile.txt"'
+    results = cmd_test(command)
     assert results['result'] is True, results['output']
 
 
-# Move test file to a new location on the SMB share
-@osx_host_cfg
-def test_55_Moving_SMB_test_file_into_a_new_directory(request):
-    depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
-    cmd = 'mkdir -p "%s/tmp" && ' % MOUNTPOINT
-    cmd += 'mv "%s/testfile.txt" ' % MOUNTPOINT
-    cmd += '"%s/tmp/testfile.txt"' % MOUNTPOINT
-    results = SSH_TEST(cmd, OSX_USERNAME, OSX_PASSWORD, OSX_HOST)
-    assert results['result'] is True, results['output']
+def test_46_verify_testfile_is_deleted_on_the_active_directory_share():
+    results = POST('/filesystem/stat/', f'{SMB_PATH}/testfile.txt')
+    assert results.status_code == 422, results.text
 
 
-# Delete test file and test directory from SMB share
-@osx_host_cfg
-def test_56_Deleting_test_file_and_directory_from_SMB_share(request):
+def test_47_delele_testfile_on_the_active_directory_share(request):
     depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
-    cmd = 'rm -f "%s/tmp/testfile.txt" && ' % MOUNTPOINT
-    cmd += 'rmdir "%s/tmp"' % MOUNTPOINT
-    results = SSH_TEST(cmd, OSX_USERNAME, OSX_PASSWORD, OSX_HOST)
+    command = fr'smbclient //{ip}/{SMB_NAME} -U {CMD_AD_USER}%{ADPASSWORD}' \
+        r' -m NT1 -c "rm testdir/testfile2.txt"'
+    results = cmd_test(command)
     assert results['result'] is True, results['output']
 
 
-@osx_host_cfg
-def test_57_Verifying_that_test_file_directory_successfully_removed(request):
-    depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
-    cmd = 'find -- "%s/" -prune -type d -empty | grep -q .' % MOUNTPOINT
-    results = SSH_TEST(cmd, OSX_USERNAME, OSX_PASSWORD, OSX_HOST)
-    assert results['result'] is True, results['output']
+def test_48_verify_testfile2_is_deleted_on_the_active_directory_share():
+    results = POST('/filesystem/stat/', f'{SMB_PATH}/testdir/testfile2.txt')
+    assert results.status_code == 422, results.text
 
 
-# Clean up mounted SMB share
-@osx_host_cfg
-def test_58_Unmount_SMB_share(request):
+def test_49_delete_testdir_on_the_active_directory_share(request):
     depends(request, ["ad_01", "ad_02", "ad_07", "ad_10"])
-    results = SSH_TEST('umount -f "%s"' % MOUNTPOINT,
-                       OSX_USERNAME, OSX_PASSWORD, OSX_HOST)
+    command = fr'smbclient //{ip}/{SMB_NAME} -U {CMD_AD_USER}%{ADPASSWORD}' \
```

Comment:
Why `fr` here when in `test_016_copy_testfile_in_testdir_on_the_active_directory_share` you're just doing `f`?

Reply:
oh, I forgot to remove it.

Reply:
That was when I was testing the smbclient failure.